target/bin/debug-getmail

```diff
@@ -5,9 +5,20 @@ source /usr/local/bin/helpers/log.sh
 # shellcheck source=../scripts/startup/setup-stack.sh
 source /usr/local/bin/setup.d/getmail.sh
 
-_setup_getmail
+# Setup getmail, even if not enabled.
+ENABLE_GETMAIL=1 _setup_getmail
 
-GETMAILDIR=/tmp/docker-mailserver/getmail
-for FILE in /etc/getmailrc.d/getmailrc*; do
-  getmail --getmaildir "${GETMAILDIR}" --rcfile "${FILE}" --dump | tail -n +6
+# Directory, where "oldmail" files are stored.
+# Getmail stores its state - its "memory" of what it has seen in your POP/IMAP account - in the oldmail files.
+GETMAIL_DIR=/var/lib/getmail
+
+# If no matching filenames are found, and the shell option nullglob is disabled, the word is left unchanged.
+# If the nullglob option is set, and no matches are found, the word is removed.
+shopt -s nullglob
+
+# Dump configuration from each RC file.
+for RC_FILE in /etc/getmailrc.d/*; do
+  echo "${RC_FILE##*/}:"
+  echo
```

Comment:
```suggestion
  echo -e "${RC_FILE##*/}:\n"
```

Reply:
I did it my way on purpose, because so it's even in the code visible clear, that there is a subject line, an empty line and then the dumped configuration.

Reply:
I don't really see it as much clearer, an empty `echo` looks odd to me with the `\n` being clearer intent of appending a line break.

I'm not fussed either way, all my suggestions are optional :)